docs/HTML.md

```diff
@@ -141,6 +142,20 @@ pdf.output("html_dd_indented.pdf")
     + `<th>`: heading cells (with `align`, `bgcolor`, `width` attributes)
     * `<td>`: cells (with `align`, `bgcolor`, `width`, `rowspan`, `colspan` attributes)
 
+### Page breaks
+_New in [:octicons-tag-24: 2.7.10](https://github.com/py-pdf/fpdf2/blob/master/CHANGELOG.md)_
+
+Page breaks can be triggered manually using the [break-before](https://developer.mozilla.org/en-US/docs/Web/CSS/break-before) or [break-after](https://developer.mozilla.org/en-US/docs/Web/CSS/break-after) CSS properties.
```

Comment:
I think "explicitly" would be better than "manually" here. It's triggered by an instruction in the data after all, not by direct user action.

Reply:
Alright!